src/Shared/Buffers.MemoryPool/PinnedBlockMemoryPoolMetrics.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+#nullable enable
+
+namespace System.Buffers;
+
+internal sealed class PinnedBlockMemoryPoolMetrics
+{
+    // Note: Dot separated instead of dash.
```

Comment:
Is that uncommon?

@JamesNK

Reply:
No, it's not uncommon for a meter name to use dots. I think at one point the kestrel event source used dashes, but that's very old.

Comment doesn't seem nessessary